### DISCUSSION TITLE
How do I implement caps word in VIAL?

### DISCUSSION THREAD

**Discussion by spencerlundgren** [vial-kb/vial-gui#279] [Q&A]
I can see [here](https://github.com/vial-kb/vial-qmk/commit/e648b84da3db022ba308163f21c3cf8165c45e09) that `caps word` is in quantum.c, however I cannot find it on the VIAL quantum page. Where do I  find the `caps word` function?
Thank you in advance for your assistance!

**Comment by xyzz** [comment]
Currently the keycode for Caps Word isn't exposed in the GUI, however you're able to enter it by enabling the feature in the firmware configuration and then using Vial "any key" dialog with the key code of 0x7C73.

**Reply by spencerlundgren** [reply]
Ok, so just to clarify:  I need to download qmk firmware, and change the configuration to allow for Caps Word. Build, then re-flash the keyboard, then add it in Vial with the any key with the key code you mentioned? Is there a step in there to make the qmk firmware compatible with vial?
Sorry for the noob question. The keyboard I have came with vial ready firmware so I haven't actually flashed it with any new firmware yet.
